catalogue_pipeline/ingestor/src/test/scala/uk/ac/wellcome/platform/ingestor/services/WorkIndexerTest.scala

```diff
@@ -165,4 +241,17 @@ class WorkIndexerTest
     val workIndexer = new WorkIndexer(elasticClient = elasticClient)
     testWith(workIndexer)
   }
+
+  private def indexWork(work: IdentifiedBaseWork, indexName: String, workIndexer: WorkIndexer) = {
```

Comment:
You can lose the braces; otherwise this is a really good helper.

👍 for this approach